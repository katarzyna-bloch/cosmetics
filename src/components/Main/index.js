import styled from 'styled-components'

const Main = styled.main`
  position: relative;
  display: flex;
  justify-content: center;
  flex: 1;
  padding-bottom: 15px;
`

export default Main
