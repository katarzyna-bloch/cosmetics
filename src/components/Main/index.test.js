import { shallow } from "enzyme"
import toJson from "enzyme-to-json"
import Main from "."

it("should render correctly", () => {
  const wrapped = shallow(<Main />)
  expect(toJson(wrapped)).toMatchSnapshot()
})
