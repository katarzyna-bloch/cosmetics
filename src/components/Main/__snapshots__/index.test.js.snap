// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<main
  className="sc-bdvvaa krsIXp"
/>
`;
